src/renderer/components/swap/Swap.tsx

```diff
@@ -207,10 +207,13 @@ export const Swap = ({
               { ...maxAmount }
             : baseAmount(targetAmount.amount(), maxAmount.decimal)
         ),
+        // Provide alt value ONLY in case there is no unlocked wallet
+        // In other case missing oAssetWB means user does not have appropriate balance
+        O.alt(() => (isLocked(keystore) ? O.some(targetAmount) : O.none)),
```

Comment:
What about not imported wallet?

Reply:
yep. `isLocked` also [check for hasImportedKeystore](https://github.com/thorchain/asgardex-electron/blob/master/src/renderer/services/wallet/util.ts#L26)

Reply:
Nevermind, `isLocked` already includes such a check